src/packages/user/user/workspace/components/user-workspace-access-settings/user-workspace-access-settings.element.ts

```diff
@@ -0,0 +1,122 @@
+import { UMB_USER_WORKSPACE_CONTEXT } from '../../user-workspace.context.js';
+import { html, customElement, state, css, repeat } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UserResponseModel } from '@umbraco-cms/backoffice/backend-api';
+import { UmbChangeEvent } from '@umbraco-cms/backoffice/event';
+import { UmbInputDocumentElement } from '@umbraco-cms/backoffice/document';
+import { UmbInputMediaElement } from '@umbraco-cms/backoffice/media';
+import { UmbUserGroupInputElement } from '@umbraco-cms/backoffice/user-group';
+
+@customElement('umb-user-workspace-access-settings')
+export class UmbUserWorkspaceAccessSettingsElement extends UmbLitElement {
+	@state()
+	private _user?: UserResponseModel;
+
+	#userWorkspaceContext?: typeof UMB_USER_WORKSPACE_CONTEXT.TYPE;
+
+	constructor() {
+		super();
+
+		this.consumeContext(UMB_USER_WORKSPACE_CONTEXT, (instance) => {
+			this.#userWorkspaceContext = instance;
+			this.observe(this.#userWorkspaceContext.data, (user) => (this._user = user));
```

Comment:
note about missing ControllerAlias, to make observe replace it self if this method is called again.